### PR TITLE
Added uib- Prefixes for angular-ui-bootstrap

### PR DESCRIPTION
**Pull request by mhennemeyer**

### CODE REVIEW

**Comment by ckosloski** [comment]
+1

**Comment by SylTi** [comment]
typeahead also depend on ui-bootstrap

**Comment by ymhuang0808** [comment]
May the pull request be merged? Perhaps, there are some indentation problems.

**Comment by ckosloski** [comment]
What needs to be done to get this merged?

**Comment by eugef** [comment]
Hi @ckosloski , please rebase your branch against the latest master and resolve the conflicts.

**Comment by ymhuang0808** [comment]
Should the PR be resolved by @mhennemeyer ?

**Comment by mitcht** [comment]
What do all the CSS changes do? Are they needed? Is there a way to do without them? I don't want a version change to affect how the directives look when people upgrade.

**Comment by mitcht** [comment]
Also like @eugef said, fix the conflicts.

**Comment by ymhuang0808** [comment]
It seems like that the CSS was only formatted.

File: index.html [hunk 1]
```diff
@@ -1273,13 +1273,6 @@
       </span>
     </div>
 
-    <div>
```
Comment (mitcht) [inline index.html]: This information about using an element as single should not be removed.

File: starter/angular-xeditable/js/xeditable.js [hunk 2]
```diff
@@ -857,7 +857,7 @@ function($parse, $compile, editableThemes, $rootScope, $document, editableContro
         // element wrapped by form
         if(ctrl[1]) {
           eFormCtrl = ctrl[1];
-          hasForm = true;
+          hasForm = attrs.eSingle === undefined;
```
Comment (mitcht) [inline starter/angular-xeditable/js/xeditable.js]: This is already upstream master.  Why is it in this pull req? https://github.com/vitalets/angular-xeditable/blob/master/starter/angular-xeditable/js/xeditable.js#L872

File: index.html [hunk 3]
```diff
@@ -681,18 +681,18 @@
 </p>
                       <!--script(src=dir+'/controller.js')-->
                       <script>app.controller('BsdateCtrl', function($scope) {
-  $scope.user = {
-    dob: new Date(1984, 4, 15)
-  };
+	$scope.user = {
```
Comment (mitcht) [inline index.html]: Please don't change all the indentation without making any actual code changes.  This just increases the delta count for diff.

File: starter/angular-xeditable/js/xeditable.js [hunk 4]
```diff
@@ -1058,6 +1058,7 @@ angular.module('xeditable').factory('editableFormController',
       if (this.$visible) {
         editable.catchError(editable.show());
       }
+      editable.catchError(editable.setWaiting(this.$waiting));
```
Comment (mitcht) [inline starter/angular-xeditable/js/xeditable.js]: Have you pulled on a different branch? This change (i don't think) is meant to be in this pull req.

**Pull request closed without merging**